lib/src/trie/minimize_proof.rs

```diff
@@ -0,0 +1,82 @@
+use hashbrown::HashSet;
+use itertools::Itertools;
```

Comment:
```suggestion
use itertools::Itertools as _;
```

Nit-pick, but I prefer